src/main/java/com/arpnetworking/clusteraggregator/configuration/ClusterAggregatorConfiguration.java

```diff
@@ -661,8 +663,8 @@ public boolean validateAggregatorLivelinessTimeout(final Duration aggregatorLive
         @NotNull
         private Duration _reaggregationTimeout = Duration.ofMinutes(1);
         @NotNull
-        @ValidateWithMethod(methodName = "validateAggregatorLivelinessTimeout", parameterType = Duration.class)
-        private Duration _aggregatorLivelinessTimeout = _reaggregationTimeout.multipliedBy(2);
+        @ValidateWithMethod(methodName = "validateAggregatorLivelinessTimeout", parameterType = Optional.class)
+        private Optional<Duration> _aggregatorLivelinessTimeout = Optional.empty();
```

Comment:
Nit: Shouldn't be an optional on the builder. The optional wrapping is done in the ctor of the class being built.

However, I am going to ask why this isn't default behavior with a reasonable default in-code value? (e.g. 10min)

Reply:
I can just use a default. I'm not actually sure how much this code is being used outside of DBX, I wanted to avoid surprise defaults for anyone else.

Reply:
I think reaping unused actors and freeing memory will be a pleasant surprise for all users of this stack.